src/features/personDetails/PersonDetailsPage/Person/index.js

```diff
@@ -7,32 +7,32 @@ import { selectCredits, selectPersonDetails } from "../../personDetailsSlice";
 import { DetailsWrapper } from "./styled";
 
 const Person = () => {
-  const personDetails = useSelector(selectPersonDetails);
-  const personCredits = useSelector(selectCredits);
+    const personDetails = useSelector(selectPersonDetails);
+    const personCredits = useSelector(selectCredits);
 
-  return (
-    <Wrapper>
-      <DetailsWrapper>
-        <DetailsTile
-          poster={personDetails.profile_path}
-          title={personDetails.name}
-          dateBirth={personDetails.birthday}
-          placeBirth={personDetails.place_of_birth}
-          description={personDetails.biography}
-        />
-        <Title title={`Movies - cast (${personCredits.cast.length})`} />
-        <MovieTiles
-          movies={personCredits.cast}
-          genres={personCredits.cast.genres_ids}
-        />
-        <Title title={`Movies - crew (${personCredits.crew.length})`} />
-        <MovieTiles
-          movies={personCredits.crew}
-          genres={personCredits.crew.genres_ids}
-        />
-      </DetailsWrapper>
-    </Wrapper>
-  );
+    return (
+        <Wrapper>
+            <DetailsWrapper>
+                <DetailsTile
+                    poster={personDetails.profile_path}
+                    title={personDetails.name}
+                    dateBirth={personDetails.birthday}
+                    placeBirth={personDetails.place_of_birth}
+                    description={personDetails.biography}
+                />
+                <Title title={personCredits.cast.length > 0 ? `Movies - cast (${personCredits.cast.length})` : ""} />
```

Comment:
Bardziej bym to widział jako { personCredits.cast.length && <Title  title={`Movies - cast (${personCredits.cast.length})` } /> }.  Albo coś na tej zasadzie, żeby w razie braku filmów tytuł w ogóle się nie renderował. Obecnie wyrenderuje się puste h2.